demo/Demo/Textfields.elm

```diff
@@ -224,6 +225,32 @@ textfields model =
         ]
        """
     )
+  , ( "Expandable textfield"
+    , Textfield.render Mdl [11] model.mdl
+        [ Textfield.label "Expandable"
+        , Textfield.floatingLabel
+        -- NOTE: The id's must match
```

Comment:
Can't we make a single property that sets both? So that you can't accidentally forget one or mismatch them?

E.g., `expandable` overrides the id set in inner?